test/over_react_redux/connect_flux_test.dart

```diff
@@ -30,27 +30,27 @@ import 'fixtures/store.dart' as redux_store;
 
 main() {
   group('connectFlux', () {
-    UiFactory<ConnectFluxCounterProps> ConnectedCounter;
-    TestJacket<ConnectFluxCounterComponent> jacket;
+    UiFactory<ConnectFluxCounterProps>? ConnectedCounter;
```

Comment:
Is this just nullable because it gets set to null in the `setUp`? Could we remove that part of the setup and remove all the awkward factory dual-invocations that are preceded by a `!` in this file?

(Same question on `connect_test.dart`)

Reply:
> Is this just nullable because it gets set to null in the setUp? Could we remove that part of the setup and remove all the awkward factory dual-invocations that are preceded by a ! in this file?

Yup, exactly.

This is one of the cases where making the variable `late` and non-nullable didn't seem worth it.

Since it's assigned in each `test` and not in a `setUp`, nulling it out before each test (or after each test) helps ensure that there's no pollution between tests (e.g., a test forgets to assign ConnectedCounter, and it runs using the previous test's setup).

Oh, but in this case... it looks like having a shared variable actually isn't necessary, and I can just make them all local variables. I'll do that instead!

Reply:
Addressed in 1b8eea5fdcda166908347a7748924868d9f02bff